src/search/FindBar.js

```diff
@@ -444,6 +444,7 @@ define(function (require, exports, module) {
         this.searchField = new QuickSearchField(searchFieldInput, {
             verticalAdjust: searchFieldInput.offset().top > 0 ? 0 : this._modalBar.getRoot().outerHeight(),
             maxResults: 20,
+            shouldHighLightFirstIndex: false,
```

Comment:
I thought we should keep the highlighting for first element as default, and if someone wants to explicitly remove this he can pass false.